clicommand/bootstrap.go

```diff
@@ -222,6 +222,7 @@ var BootstrapCommand = cli.Command{
 			Name:   "shell",
 			Usage:  "The shell to use to interpret build commands",
```

Comment:
Should we mention "This defaults to our best guess at what makes sense based on the current operation system" or something? Just thinking about what we copy/paste into the docs for this `--help` output.

Reply:
Yeah, makes sense.

Reply:
Oh, this is the bootstrap cli command. Did you mean here or the agent start command?

Reply:
> Did you mean here or the agent start command?

Oh yeah, I meant that!

Reply:
"The shell to use to interpret build commands. Defaults to bash or cmd.exe depending on your operating system."

Reply:
👍

Reply:
Or maybe "The shell commands used to interpret build commands, e.g /bin/bash -e -c"

Reply:
Yeah that works as well